### DISCUSSION TITLE
1 SSD e Varios HD's

### DISCUSSION THREAD

**Discussion by decsousa** [Chia-Network/chia-blockchain#2591] [Q&A]
Olá,

Gostaria de tirar uma dúvida sobre performance.
Se eu tiver 1 SSD de 240GB e varios HD's totalizando 16TB tenho um bom desempenho ou preciso de um SSD maior ou até mais que um SSD?

Obrigado.

**Comment by athoszxz** [comment]
O de 240GB não funcionou pra mim, tive que usar o de 447GB pra conseguir plotear no K32. O HD pra minerar depois não precisa ser rápido você só tem que ver se você tem portas satas e USB (no caso de HD externo) suficientes para todos.
